cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1774,14 +1774,21 @@ public void checkCustomCaSecret(CertificateAuthority ca, Secret certSecret, Secr
                                 Secret certSecret = result.result();
 
                                 if (certSecret != null) {
-                                    try {
-                                        X509Certificate cert = Ca.cert(certSecret, customCertSecret.getCertificate());
-                                        byte[] signature = MessageDigest.getInstance("SHA-256").digest(cert.getEncoded());
-                                        thumbprintPromise.complete(Base64.getEncoder().encodeToString(signature));
-                                    } catch (CertificateEncodingException | NoSuchAlgorithmException e) {
-                                        log.warn("{}: Failed to get certificate signature of {} from Secret {}.", reconciliation, customCertSecret.getCertificate(), customCertSecret.getSecretName());
-                                        thumbprintPromise.fail(new RuntimeException("Failed to get certificate signature of " + customCertSecret.getCertificate() + " from Secret " + certSecret.getMetadata().getName(), e));
-                                    }
+                                    if (!certSecret.getData().containsKey(customCertSecret.getCertificate())) {
+                                        log.warn("{}: Secret {} does not contain certificate under the key {}.", reconciliation, customCertSecret.getSecretName(), customCertSecret.getCertificate());
```

Comment:
Won't the logging here and the failure of the `thumbprintPromise` with the same message result in the message being logged twice?

Reply:
It will: https://pastebin.com/8wpCi6VZ

Reply:
Well it's pointless logging such things twice, so I'd drop the log stmts here and let the global handler log it.

Reply:
@scholzj ^ What do you think?

Reply:
Sure, fine with me to remove them.